tests/css-epub-text-orientation/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,17 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>EPUB CSS Text Orientation</title>
+  </head>
+  <body>
+    <h1>-epub-text-orientation Tests</h1>
+
+    <p style="margin: 1em; -epub-text-orientation: mixed">1. これはテストです。This test passes if the English text appears to be turned 90 degrees while the Japanese text is vertical.</p>
```

Comment:
Does writing-mode not also need to be set?
https://developer.mozilla.org/en-US/docs/Web/CSS/text-orientation